app/src/main/java/org/simple/clinic/login/OtpSmsReceiver.kt

```diff
@@ -31,6 +31,14 @@ class OtpSmsReceiver : BroadcastReceiver() {
   @Inject
   lateinit var userSession: UserSession
 
+  // This is intentionally lazy so that the country specific retrofit endpoint
+  // is not instantiated until the SMS is received.
+  @Inject
+  lateinit var loginUserWithOtp: dagger.Lazy<LoginUserWithOtp>
```

Comment:
This is necessary for now because the `OtpSmsReceiver` is registered in the Android Manifest and we do not want to instantiate this class until the country is selected. We should figure out a better way to fix this.

Reply:
This should probably get fixed when we create the app and country scope right?

Reply:
That's the plan!